lib/std/math.zig

```diff
@@ -406,6 +406,48 @@ pub const min3 = @compileError("deprecated; use @min instead");
 pub const max3 = @compileError("deprecated; use @max instead");
 pub const ln = @compileError("deprecated; use @log instead");
 
+/// Odd sawtooth function
+/// ```
+///         |
+///      /  | /    /
+///     /   |/    /
+///  --/----/----/--
+///   /    /|   /
+///  /    / |  /
+///         |
+/// ```
+/// Limit x to the half-open interval [-r, r).
+pub fn wrap(x: anytype, r: anytype) @TypeOf(x) {
+    return @mod(x + r, 2 * r) - r;
+}
+test "wrap" {
```

Comment:
Could you also add a doctest like this:
```zig
test wrap {
    const limit: i32 = 180;
    // Within range
    try testing.expect(wrap(@as(i32, -75), limit) == -75);
    // Below
    try testing.expect(wrap(@as(i32, -225), limit) == 135);
    // Above
    try testing.expect(wrap(@as(i32, 361), limit) == 1);
}
```
This will make it show up in the docs and is nice to have for new additions to the standard library.

Reply:
When you write it without quotes, it shows up in the docs as a Usage Example (for an example, scroll to the bottom here: https://ziglang.org/documentation/master/std/#A;std:json.Scanner )
When you write it with quotes, it only tests it without making it show up in the docs.